components/CardSection/CategoryAnimals.js

```diff
@@ -0,0 +1,54 @@
+import { words } from "../../dummydata/words";
+import { Fragment } from "react";
+import styled from "styled-components";
+
+export default function CardSection() {
+  const allCategories = words.map((word) => word.category);
+  const singleCategories = Array.from(new Set(allCategories));
+  const animals = words.filter((word) => word.category === "animals");
```

Comment:
please check if animals is the proper name for your variable here, since you are rendering vocabularies of languages.